web/packages/build/.eslintrc.js

```diff
@@ -88,6 +88,11 @@ module.exports = {
     'import/no-unresolved': 0,
     'no-unused-vars': 'off', // disabled to allow the typescript one to take over and avoid errors in reporting
     '@typescript-eslint/no-unused-vars': ['error'],
+    'no-unused-expressions': 'off',
+    '@typescript-eslint/no-unused-expressions': [
```

Comment:
@rudream no-unused-expressions complains about this line and one below it that simply attempts to read the property. Is reading the property alone enough to force reflow?

https://github.com/gravitational/teleport/blob/b0e1f9c45955d3db6023ab1dbf4760292acf4d89/web/packages/teleport/src/Navigation/RecentHistory.tsx#L179